nifpga/session.py

```diff
@@ -129,7 +126,10 @@ def __init__(self,
         for name, bitfile_fifo in iteritems(bitfile.fifos):
             assert name not in self._fifos, \
                 "One or more FIFOs have the same name '%s', this is not supported" % name
-            self._fifos[name] = _FIFO(self._session, self._nifpga, bitfile_fifo)
+            if bitfile_fifo.is_fxp():
```

Comment:
Put this in a helper method _create_fifo to better match the register counterpart.